README.NewResourceFromTerraform.md

```diff
@@ -336,14 +375,45 @@ section under the Appendix.
 
 ## Run Tests
 
-1.  Run your newly added resource's test cases in `pkg/controller/dynamic`,
-    replacing `spannerinstance` with the name of your resource:
+1.  Set `[test-target]` to the kind (in all lower-case letters) in the following
+    command to run all the tests for your resource.
+
+    ```bash
+    go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests [test-target] -timeout 900s
+    ```
+
+    If you have multiple test cases and just want
+    to run a single test case of your resource, you can set `[test-target]` to a
+    specific test case folder name. For example:
 
     ```
-    go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests spannerinstance
+    pkg/test/resourcefixture/testdata/basic
+    ├── pubsub
+    |   ├── v1beta1
+    |       ├── pubsubsubscription
+    |           ├── basicpubsubsubscription
+    |               └── create.yaml
+    |               └── dependencies.yaml
+    |               └── update.yaml
+    |           ├── bigquerypubsubsubscription
+    |               └── create.yaml
+    |               └── dependencies.yaml
+    |               └── update.yaml
+    ```
+
+    ```bash
+    go test -v -tags=integration ./pkg/controller/dynamic/ -test.run TestCreateNoChangeUpdateDelete -run-tests basicpubsubsubscription -timeout 900s
     ```
 
-    Ensure they are successful.
+1.  If the dynamic test data contains constant placeholder
+    strings, e.g. "TEST_FOLDER_ID", you need to set them as env variables
+    first as follows:
+
+    ```bash
+       # Export the environment variables needed in the dynamic tests if you haven't done it.
+       source scripts/shared-vars.sh
```

Comment:
This is an internal only script, so not applicable here.

We can delete this since you have an example below which tells how to specify the environment variable in the same `go test` command.